pkg/network/protocols/http/ebpf_ssl.go

```diff
@@ -291,12 +293,16 @@ func (o *sslProgram) ConfigureManager(m *errtelemetry.Manager) {
 }
 
 func (o *sslProgram) ConfigureOptions(options *manager.Options) {
-
 	options.MapSpecEditors[sslSockByCtxMap] = manager.MapSpecEditor{
 		Type:       ebpf.Hash,
 		MaxEntries: uint32(o.cfg.MaxTrackedConnections),
 		EditorFlag: manager.EditMaxEntries,
 	}
+	options.MapSpecEditors[dispatcherConnectionProtocol] = manager.MapSpecEditor{
+		Type:       ebpf.Hash,
+		MaxEntries: uint32(o.cfg.MaxTrackedConnections),
+		EditorFlag: manager.EditMaxEntries,
+	}
```

Comment:
probably worth extracting this to a separate PR if this is the fix to the spinlock contention you mentioned

Reply:
https://github.com/DataDog/datadog-agent/pull/15166